libbpf-rs/src/map.rs

```diff
@@ -409,16 +426,81 @@ impl Map {
     pub fn keys(&self) -> MapKeyIter {
         MapKeyIter::new(self, self.key_size())
     }
+
+    /// Create the bpf map standalone.
+    pub fn create_map(
+        name: &str,
+        map_type: MapType,
+        key_size: u32,
+        value_size: u32,
+        max_entries: u32,
+        flags: MapCreateFlags,
+    ) -> Result<Map> {
+        let opts = libbpf_sys::bpf_map_create_opts {
+            sz: std::mem::size_of::<libbpf_sys::bpf_map_create_opts>() as libbpf_sys::size_t,
+            map_flags: flags.bits as i32,
+            btf_fd: 0,
+            btf_key_type_id: 0,
+            btf_value_type_id: 0,
+            btf_vmlinux_value_type_id: 0,
+            inner_map_fd: 0,
+            map_extra: 0,
+            numa_node: 0,
+            map_ifindex: 0,
+        };
+
+        let map_name_c = util::str_to_cstring(name)?;
+        let fd = unsafe {
+            libbpf_sys::bpf_map_create(
+                map_type.clone() as u32,
+                map_name_c.as_ptr(),
+                key_size,
+                value_size,
+                max_entries,
+                &opts,
+            )
+        };
+
+        if fd < 0 {
+            return Err(Error::System(fd));
+        }
+
+        Ok(Map {
+            fd,
+            name: name.to_string(),
+            ty: map_type as u32,
+            key_size,
+            value_size,
+            ptr: std::ptr::null_mut(),
+        })
+    }
 }
 
 #[rustfmt::skip]
 bitflags! {
     /// Flags to configure [`Map`] operations.
     pub struct MapFlags: u64 {
-	const ANY      = 0;
-	const NO_EXIST = 1;
-	const EXIST    = 1 << 1;
-	const LOCK     = 1 << 2;
+        const ANY      = 0;
+        const NO_EXIST = 1;
+        const EXIST    = 1 << 1;
+        const LOCK     = 1 << 2;
```

Comment:
I'm not sure what the change here is - this looks the same to me. Maybe some whitespace difference? Would you mind reverting any whitespace changes here so it doesn't show up as modified?